classes/05class/exercises/c05-3m01/obrientimothya/Makefile

```diff
@@ -0,0 +1,13 @@
+DOCKER=docker-compose run --rm
+
+build:
+	docker build -t obrientimothya/aws-cli:2.0.0 .
+.PHONY: build
+
+create_bucket:
+	$(DOCKER) aws bash scripts/create_bucket.sh
```

Comment:
Minor: you can create `_create_bucket` targets to separate technology-specific commands and leave the higher-level ones just for `make`. (Bash is not included in the 3M patterns, but it is commonly used. It could be any shell or application)

https://3musketeers.io/docs/patterns.html#make